### ISSUE TITLE
Add link to Github Project

### ISSUE THREAD

**Issue by julian-poidevin** [julian-poidevin/MBPMid2010_GPUFix#3]
Maybe in an `About` menu.

**Comment by julian-poidevin** [comment]
See example below : 

```cpp
void PulsepickerWindow::showEvent(QShowEvent *event)
{
    QSettings settings(QApplication::applicationDirPath() + globalSettingsName, QSettings::IniFormat);
    settings.beginGroup("Stylesheet");
    QString logoPath = settings.value("Logo", "").toString();
    settings.endGroup();

    if(logoPath.isEmpty())
    {
        logoPath = ":/Elements/Logo_alphanov.png";
    }

    QPixmap pix = QPixmap (logoPath);
    int width = this->ui->labelLogo->width();
    int height = this->ui->labelLogo->height();

    pix = pix.scaled(width, height,Qt::KeepAspectRatio, Qt::SmoothTransformation);
    this->ui->labelLogo->setPixmap(pix);

    event->accept();
}
```

**Comment by julian-poidevin** [comment]
Fixed with release v0.2.0
